### PR TITLE
Deprecated `setIsUiVisible` and `getIsUiVisible` of `UiFramework`

### PR DESCRIPTION
**Pull request by MBudreviciusBentley**
## Changes

Deprecate `setIsUiVisible` and `getIsUiVisible` of `UiFramework`

## Testing

N/A

### CODE REVIEW

File: ui/appui-react/src/appui-react/UiFramework.ts [hunk 1]
```diff
@@ -657,10 +657,12 @@ export class UiFramework {
     return useGlobalStore.getState().viewState;
   }
 
+  /** @deprecated in 4.17.0. */
   public static getIsUiVisible() {
     return UiFramework.visibility.isUiVisible;
   }
 
+  /** @deprecated in 4.17.0. */
```
Comment (GerardasB) [inline ui/appui-react/src/appui-react/UiFramework.ts:665]: Needs a replacement message.
```suggestion
  /** @deprecated in 4.17.0. Use {@link UiFramework.visibility.isUiVisible} instead. */
```

File: docs/changehistory/NextVersion.md [hunk 2]
```diff
@@ -4,16 +4,21 @@ Table of contents:
...
 
 ## @itwin/appui-react
 
 ### Changes
 
 - Allow to set the available snap modes in `SnapModeField` component. [#974](https://github.com/iTwin/appui/pull/974)
 
+### Deprecations
+
+- Deprecated `UiFramework.setIsUiVisible` and `UiFramework.getIsUiVisible`. [#1023](https://github.com/iTwin/appui/pull/1023)
```
Comment (GerardasB) [inline docs/changehistory/NextVersion.md:20]:
```suggestion
- Deprecated `UiFramework.setIsUiVisible` and `UiFramework.getIsUiVisible`. Use `UiFramework.visibility.isUiVisible` instead. [#1023](https://github.com/iTwin/appui/pull/1023)
```

File: ui/appui-react/src/appui-react/UiFramework.ts (hunk 1 above)
Comment (MBudreviciusBentley) [reply ui/appui-react/src/appui-react/UiFramework.ts]: Done.

File: ui/appui-react/src/appui-react/UiFramework.ts [hunk 3]
```diff
@@ -657,10 +657,12 @@ export class UiFramework {
     return useGlobalStore.getState().viewState;
   }
 
+  /** @deprecated in 4.17.0. */
```
[the remark this answers is not in the document]
Comment (MBudreviciusBentley) [reply ui/appui-react/src/appui-react/UiFramework.ts]: Done.

File: docs/changehistory/NextVersion.md (hunk 2 above)
Comment (MBudreviciusBentley) [reply docs/changehistory/NextVersion.md]: Done.

**Pull request merged**